purchase_invoice_plan/models/purchase.py

```diff
@@ -206,79 +206,78 @@ def action_invoice_create(self):
                 last_installment = max(installments)
                 for installment in installments:
                     # Getting invoice plan for each installment
-                    plan_lines = plan_obj.search(
+                    blines = plan_obj.search(
```

Comment:
I think `plan_lines` is not needed, is it?
With the old code, to gives error for case > 1 PO line to have > 1 installment.